docs/install/install.md

```diff
@@ -19,38 +19,113 @@ limitations under the License.
 -->
 {% include JB/setup %}
 
-## Zeppelin Installation
-Welcome to your first trial to explore Zeppelin!
+# Quick Start
+Welcome to your first trial to explore Apache Zeppelin! 
+This page will help you to get started with Apache Zeppelin. Here is the list of this page. 
 
-In this documentation, we will explain how you can install Zeppelin from **Binary Package** or build from **Source** by yourself. Plus, you can see all of Zeppelin's configurations in the [Zeppelin Configuration](install.html#zeppelin-configuration) section below.
+* [Installation](#installation)
+  * [Downloading Binary Package](#downloading-binary-package)
+  * [Building from Source](#building-from-source)
+* [Starting Apache Zeppelin with Command Line](#starting-apache-zeppelin-with-command-line)
+  * [Start Zeppelin](#start-zeppelin)
+  * [Stop Zeppelin](#stop-zeppelin)
+  * [(Optional) Start Apache Zeppelin with a service manager](#optional-start-apache-zeppelin-with-a-service-manager)
+* [What is the next?](#what-is-the-next)
+* [Apache Zeppelin Configuration](#apache-zeppelin-configuration)
 
-### Install with Binary Package
+## Installation
 
-If you want to install Zeppelin with latest binary package, please visit [this page](http://zeppelin.apache.org/download.html).
+Apache Zeppelin officially tested on the below environment. 
 
-### Build from Zeppelin Source
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Value</th>
+  </tr>
+  <tr>
+    <td>Oracle JDK</td>
+    <td>1.7 <br /> (set <code>JAVA_HOME</code>)</td>
+  </tr>
+  <tr>
+    <td>OS</td>
+    <td>Mac OSX <br /> Ubuntu 14.X <br /> CentOS 6.X <br /> Windows 7 Pro SP1</td>
+  </tr>
+</table>
+
+There are two options to install Apache Zeppelin on your machine. One is [downloading prebuild binary package](#downloading-binary-package) from the archive. 
+You can download not only the latest stable version but also the older one if you need. 
+The other option is [building from the source](#building-from-source).
+Although it can be unstable somehow since it is on development status, you can explore newly added feature and change it as you want.
+
+### Downloading Binary Package
+
+If you want to install Apache Zeppelin with a stable binary package, please visit [Apache Zeppelin download Page](http://zeppelin.apache.org/download.html). 
+After unpacking, jump to [Starting Apache Zeppelin with Command Line](#starting-apache-zeppelin-with-command-line) section.
+
+### Building from Source
+If you want to build from the source, there are more requirements. 
+
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Value</th>
+  </tr>
+  <tr>
+    <td>Git</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Maven</td>
+    <td>3.1.x or higher</td>
+  </tr>
+</table>
+
+If you don't have the above requirements yet, please check [Before Build](https://github.com/apache/zeppelin/blob/master/README.md#before-build) section and follow step by step.
```

Comment:
If you don't have it installed yet, please check... and follow step by step instructions from there.